boards/shields/st_b_lcd40_dsi1_mb1166/boards/stm32h747i_disco_m7.overlay

```diff
@@ -10,6 +10,16 @@
 	chosen {
 		zephyr,display = &ltdc;
 	};
+
+	lvgl_pointer {
+		compatible = "zephyr,lvgl-pointer-input";
+		input = <&ft5336>;
+		invert-y;
+	};
+};
+
+&sdram2 {
+	/delete-property/ zephyr,memory-attr;
```

Comment:
`zephyr,memory-attr` on sdram2 might be used by other applications. Also adding custom linker scripts is old fashioned. Can't you use `mem_attr_get_regions` API for the `lvgl_buf` section ?

Reply:
`zephyr,memory-attr` also work, should select `no_cache` version.

Reply:
It would be great if you can use it.

Reply:
I can try it latter today.